docs/release-notes.md

```diff
@@ -8,6 +8,10 @@ nav_order: 9
 
 ### Breaking changes
 
+- The dracut module is not automatically included in initramfs images anymore.
+  Distributions need to explicitly request inclusion, either by ensuring that
+  it is pulled in by another module, for instance the module providing distro
+  integration and customization, or by installing a dracut configuration file.
```

Comment:
These release notes are usually not so long. How about just:

> Only include dracut module in initramfs if requested

?

Reply:
IMO for breaking changes there should be some explanation on how to deal with them. An integrator might not know directly what "requested" in this context means.

Reply:
We can add more information in the `distributor-notes.md` (which gets rendered [here](https://coreos.github.io/ignition/distributor-notes/)) and refer to it from the release notes. E.g.

> Only include dracut module in initramfs if requested (see distributor notes for details)

Reply:
Done.